fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2338,7 +2340,7 @@ public void replayCreateTable(CreateTableInfo info) {
             }
         }
 
-        if (table.isCloudNativeTableOrMaterializedView()) {
+        if (table.isCloudNativeTableOrMaterializedView() && info.getStorageVolumeId() != null) {
```

Comment:
is it good enough, is there any other places that calls replay bind db or table storage volume?

Reply:
yes. I have checked.